llvm/lib/LTO/LTOBackend.cpp

```diff
@@ -79,11 +79,6 @@ namespace llvm {
 extern cl::opt<bool> NoPGOWarnMismatch;
 }
 
-static cl::opt<bool>
```

Comment:
How comes this disappeared? Was it moved elsewhere in another commit perhaps?

Reply:
LLVM 14 didn't like that we had two of them. The other one that I kept lives here: https://github.com/ykjit/ykllvm/blob/ykjit/13.x/llvm/lib/CodeGen/TargetPassConfig.cpp#L247

Reply:
A possible explanation is that those two weren't in the same namespace previously which is why we needed both, but LLVM 14 changed the namespaces so now they were and thus LLVM wasn't happy.

Reply:
If there is duplication, then deleting one is certainly correct!